interop/Cargo.toml

```diff
@@ -38,6 +38,8 @@ tempfile = { version = "3.15" }
 # Terminal support
 xshell = "0.2"
 spinoff = { version = "0.8", features = ["aesthetic"], default-features = false }
+serde = { version = "1.0.217", features = ["derive"] }
```

Comment:
Why are we using this specific version of `serde`, and not just using the one from workspace?

Reply:
No reason, was added by the IDE